addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -284,8 +284,8 @@ class StudyService extends Service {
   }
 
   async create(requestContext, rawStudyEntity) {
-    if (!(isInternalResearcher(requestContext) || isAdmin(requestContext))) {
-      throw this.boom.forbidden('Only admin and internal researcher are authorized to create studies.', true);
+    if (!isAdmin(requestContext)) {
```

Comment:
Just curious because I wasn't aware of this - Are researchers not allowed to create studies anymore (even in the "My Study" category)?

Reply:
Oh please don't review this PR yet. It's just in draft stage for reference purposes. There's a potential to enable this feature through a config change. More info in GALI-1341